packages/core/src/tools/utils.ts

```diff
@@ -209,11 +209,21 @@ export function includes(candidate: string | unknown[], search: any) {
   return candidate.indexOf(search) !== -1
 }
 
-export function arrayFrom<T>(arrayLike: ArrayLike<T>): T[] {
+export function arrayFrom<T>(arrayLike: ArrayLike<T> | Set<T>): T[] {
+  if (Array.from) {
+    return Array.from(arrayLike)
+  }
+
   const array = []
-  for (let i = 0; i < arrayLike.length; i++) {
-    array.push(arrayLike[i])
+
+  if (arrayLike instanceof Set) {
+    arrayLike.forEach((item) => array.push(item))
+  } else {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice! Things we should do to support IE...

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Remove the `setToArray` function from this file and replace usages with `arrayFrom`